src/draw/lv_draw_vector.h

```diff
@@ -279,6 +284,13 @@ void lv_vector_dsc_set_fill_rule(lv_vector_dsc_t * dsc, lv_vector_fill_t rule);
  */
 void lv_vector_dsc_set_fill_image(lv_vector_dsc_t * dsc, const lv_draw_image_dsc_t * img_dsc);
 
+/**
+ * Set the coordinate system of the fill image
+ * @param dsc              pointer to a vector graphic descriptor
+ * @param utils            the coordinate system to be set in lv_vector_fill_utils_t format
+ */
+void lv_vector_dsc_set_fill_pattern_utils(lv_vector_dsc_t * dsc, const lv_vector_fill_utils_t utils);
```

Comment:
Should fill_units also support gradient, so use `lv_vector_dsc_set_fill_units`?

Reply:
Done.